docs/dev/table/sql/create.md

```diff
@@ -127,14 +127,21 @@ CREATE TABLE [catalog_name.][db_name.]table_name
   (
     { <column_definition> | <computed_column_definition> }[ , ...n]
     [ <watermark_definition> ]
+    [ <out_of_line_constraint> ][ , ...n]
   )
   [COMMENT table_comment]
   [PARTITIONED BY (partition_column_name1, partition_column_name2, ...)]
   WITH (key1=val1, key2=val2, ...)
   [ LIKE source_table [( <like_options> )] ]
    
 <column_definition>:
-  column_name column_type [COMMENT column_comment]
+  column_name column_type [ <in_line_constraint> ] [COMMENT column_comment]
+  
+<in_line_constraint>:
+  [CONSTRAINT constraint_name] PRIMARY KEY [[NOT] ENFORCED]
```

Comment:
Let's force to use `NOT ENFORCED` here, because this is the only mode we support. Otherwise, the syntax will confuse users.